public/app/frontend/src/components/navigation-secondary/navigation-secondary.html.twig

```diff
@@ -22,58 +23,87 @@ example:
             {
                 url: '#',
                 label: 'Click here'
+            },
+            {
+                url: '#',
+                label: 'Click here',
+                children: [
+                    {...}
+                ],
+                expanded: true
             }
         ]
     }%}
 #}
 
 {% import _self as menus %}
 
+{# TODO handle question marks in IDs #}
+
 {# Generate menus from a list of links. Called recursively to generate child links #}
-{% macro menu_links(items, menu_level, menu_id, articleId) %}
+{% macro menu_links(items, menu_level, menu_id, articleId, expanded) %}
     {% import _self as menus %}
+
     {% if items %}
         {% if menu_level == 0 %}
-<ul class="navigation-secondary__list">
-    <li class="navigation-secondary__skip-to-article navigation-secondary__list-item navigation-secondary__list-item--level-{{ menu_level }}">
-        <div class="navigation-secondary__list-item-wrapper">
-            <a class="navigation-secondary__link" href="{{ articleId }}">Skip to main content</a>
-        </div>
-    </li>
-    {% else %}
-    <ul id="{{ menu_id }}" class="navigation-secondary__sublist navigation-secondary__list--level-{{ menu_level }} {{ item.active ? " navigation-secondary__sublist--open" : "" }}">
-        {% if item.children %}
-            <li class="navigation-secondary__list-item navigation-secondary__list-item--level-{{ menu_level }}{{ item.active ? " navigation-secondary__list-item--active" : "" }}">
-                <div class="navigation-secondary__list-item-wrapper navigation-secondary__show-all">
-                    <a class="navigation-secondary__link" role="link" {{ item.active ? "aria-disabled='true'" : "href='#{ item.link }'" }}>
-                        All {{ item.label }} content
-                    </a>
-                </div>
-            </li>
-        {% endif %}
```

Comment:
This block is removed because there is no `item` variable at this point, and no styling for the `navigation-secondary__show-all` class.